homeassistant/components/wallbox/__init__.py

```diff
@@ -78,12 +83,19 @@ def _get_data(self) -> dict[str, Any]:
             data[CONF_MAX_CHARGING_CURRENT_KEY] = data[CONF_DATA_KEY][
                 CONF_MAX_CHARGING_CURRENT_KEY
             ]
+            data[CONF_LOCKED_UNLOCKED_KEY] = data[CONF_DATA_KEY][
+                CONF_LOCKED_UNLOCKED_KEY
+            ]
 
             return data
 
         except requests.exceptions.HTTPError as wallbox_connection_error:
             raise ConnectionError from wallbox_connection_error
```

Comment:
Side note: We don't seem to catch `ConnectionError` anywhere. In the coordinator update method we should raise `UpdateFailed` for expected errors like connection error.